src/dodal/devices/focusing_mirror.py

```diff
@@ -107,30 +121,48 @@ def voltage_channels(self) -> list[MirrorVoltageDevice]:
         ]
 
 
-class FocusingMirror(Device):
+class FocusingMirror(StandardReadable):
     """Focusing Mirror"""
 
-    def __init__(self, bragg_to_lat_lut_path, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(
+        self, name, prefix, bragg_to_lat_lut_path=None, x_suffix="X", y_suffix="Y"
+    ):
         self.bragg_to_lat_lookup_table_path = bragg_to_lat_lut_path
+        self.yaw_mrad = Motor(prefix + "YAW")
+        self.pitch_mrad = Motor(prefix + "PITCH")
+        self.roll_mrad = Motor(prefix + "ROLL")
+        self.x_mm = Motor(prefix + x_suffix)
+        self.y_mm = Motor(prefix + y_suffix)
+        self.jack1_mm = Motor(prefix + "Y1")
+        self.jack2_mm = Motor(prefix + "Y2")
+        self.jack3_mm = Motor(prefix + "Y3")
+        self.translation1_mm = Motor(prefix + "X1")
+        self.translation2_mm = Motor(prefix + "X2")
+
+        self.type, _ = soft_signal_r_and_backend(MirrorType, MirrorType.SINGLE)
```

Comment:
Happy with this as a preferred way to include config for now, will do the same with Undulator and DCM (although in a separate PR to #461)